forte/data/multi_pack.py

```diff
@@ -168,7 +174,15 @@ def _validate(self, entry: EntryType) -> bool:
 
     # TODO: get_subentry maybe useless
     def get_subentry(self, pack_idx: int, entry_id: int):
```

Comment:
I believe we need docstring for the arguments

Reply:
docstring added for the list index to pack_id access (of packs in lists) change for multi pack